src/index.js

```diff
@@ -17,18 +17,18 @@ gisida.version = require('../package.json').version;
 gisida.initStore = require('./store/initStore').default;
 gisida.loadLayers = require('./store/initStore').loadLayers;
 gisida.reducerRegistry = require('./store/reducerRegistry').default;
-gisida.createMapReducer = require('./store/reducers/app').createMapReducer;
+gisida.createMapReducer = require('./store/reducers/createMapReducer/createMapReducer').createMapReducer;
```

Comment:
@kelvin-muchiri Is it possible to get the `createMapReducer` (and `MAP` on line 9) [here through the `createMapReducer`'s `index.js`](https://github.com/onaio/gisida/blob/master/src/store/reducers/createMapReducer/index.js) file if we actually import the modules (instead of exporting a string).

Reply:
> @kelvin-muchiri Is it possible to get the `createMapReducer` (and `MAP` on line 9) [here through the `createMapReducer`'s `index.js`](https://github.com/onaio/gisida/blob/master/src/store/reducers/createMapReducer/index.js) file if we actually import the modules (instead of exporting a string).

I am not sure whether I understood correctly but I made the change.

Reply:
@kelvin-muchiri I'm try to make sense of the way Gisida `src/index.js` is referencing the different files in the `createMapReducer` module. Basically I think the files in the `createMapReducer` directory should be referenced via `createMapReducer/index.js` to make the `createMapReducer` functionality more portable (aka modular).

And actually, it may make more sense to change `src/store/reducers/createMapReducer/` to be `src/store/reducers/map` because the reducers contained seem to be all about the map. Consider:

### src/reducers/map/createMapReducer.js
While we can define things in different files in a directory...
```
export createMapReducer(mapId) { ... } // A - defines / exports the function to create a Map reducer

const MAP = createMapReducer('map-1') // B - defines the default Map reducer
export default MAP; // B - exports the default Map reducer as the default export
```

### src/reducers/map/index.js
...we should make them available from outside the directory in one place...
```
import MAP, { createMapReducer } from './createMapReducer';
export { createMapReducer }; // A - exports the function to create a Map reducer
export default MAP; // B - exports the default Map reducer as the default export
```
### src/index.js
...so the whole directory can be treated as a single module.
```
// here, both default Map reducer and function to create a Map reducer
// are referenced via src/reducers/map/index.js
import MAP, { createMapReducer } from `./store/reducers/map`;
gisida.createMapReducer = createMapReducer;
```

Reply:
@cKellyDesign I have updated